### DISCUSSION TITLE
user-guide/faqs/zotero-7-beta-versions

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#245] [文档评论区]
# user-guide/faqs/zotero-7-beta-versions

Zotero 非官方中文维护小组

https://zotero-chinese.com/user-guide/faqs/zotero-7-beta-versions

**Comment by MoYuXianR** [comment]
zotero 84版本 可以支持1w页以上的pdf，进行覆盖滚动，对于大屏更友好，便于观看和笔记

**Comment by pei458** [comment]
请问历史版本是按最新到最旧的时间顺序列的表格吗，版本号2和版本号107哪个对应最新历史版本
